hugr-passes/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## 0.1.0 (2024-05-29)
```

Comment:
The changelogs here and elsewhere don't seem to correspond to the specific crates that they belong to, which is a bit confusing; is this intentional?

Reply:
We could manually edit them, but that's going to be a burden going forward.